docs/source/guides/staking_guide.rst

```diff
@@ -237,6 +253,35 @@ the address to checksum format in geth console:
 After this step, you're finished with the Staker, and you can proceed to :ref:`ursula-config-guide`.
 
 
+Manage automatic reward restaking
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As your Ursula performs work, you can optionally enable the automatic addition of
+all rewards to your existing stake to optimize earnings.  By default this feature is disabled,
```

Comment:
Is re-staking all or nothing? Do we allow for partial re-staking of rewards?

Reply:
Is restaking enabled or disabled by default? Earlier in this commit, there is `--enable (default)`

Reply:
> Is restaking enabled or disabled by default? 

Disabled.

> Earlier in this commit, there is --enable (default)

Hmm - perhaps that's confusing.  It's meant too indicate that running `nucypher stake restake` is the same as running `nycypher stake restake --enable`, since *--enable* is on by default when running `restake`.

Reply:
> Is re-staking all or nothing? Do we allow for partial re-staking of rewards?

Hmm - no I dont think so, However you can use multiple addresses.  @szotov ?

Reply:
Yeah, right now restake is for everything if enabled